app/controllers/sign_up/completions_controller.rb

```diff
@@ -38,10 +37,6 @@ def confirm_identity_verified
       redirect_to idv_url if current_user.identity_not_verified?
     end
 
-    def confirm_selfie_captured
```

Comment:
:+1:

This should be fine to be removed since the selfie capture requirement is enforced by `OpenidConnect::AuthorizationsController` and `SamlIdpController`.